portality/settings.py

```diff
@@ -120,6 +120,29 @@
 #MAIL_MAX_EMAILS            # default None
 #MAIL_SUPPRESS_SEND         # default app.testing
 
+# ================================
+# File store
+
+STORE_IMPL = "portality.store.StoreS3"
```

Comment:
This could be dangerous to the live data - it defaults to using S3 with the live bucket. If someone accidentally uses an admin key (or one that writes) they could mess up the data.

This should be ```None``` and print a warning on start, and be set in `production.cfg` for the live server.